packages/cycle-scripts/package.json

```diff
@@ -30,12 +30,16 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
```

Comment:
wow webpack-blocks runs on 6.17.0 of babel-core and 6.2.5 of babel-loader. I knew this wasn't a good idea... :(. 
Also seems that updating is not something done on a regular base...just 3 release...my bad I didn't check this earlier